shared/common-adapters/switch-toggle.native.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import React from 'react'
+import {NativeAnimated, NativeEasing} from './native-wrappers.native'
+import * as Styles from '../styles'
+import {type Props} from './switch-toggle'
+
+class SwitchToggle extends React.PureComponent<Props, {||}> {
```

Comment:
`checkbox.native` looks like this, but is only ever blue left-aligned. it'd be cool if Checkbox used this on mobile so they looked the same. cc @cecileboucheron what do you think?

Reply:
Yeah I was wondering about that too but forgot. Thanks for bringing it up!